stream/stages/generation/tiled_workload_generation.py

```diff
@@ -165,7 +166,14 @@ def get_outer_tmap_loop_dimensions(self, node: ComputationNode) -> list[Temporal
         Returns:
             temporal loops outside of cn
         """
-        outer_loops = convert_outer_cn_loops(node.intra_core_tiling.copy(), node)
+        if contains_wildcard(node.inter_core_tiling):
```

Comment:
I would check each tuple individually for a wildcard and only filter out those. That way atleast this part of the code is robust for future changes where we can partially fix the inter_core_tiling/intra_core_tiling and leave part up to another optimizer to set the correct tiling later?